net/dnsviz/Portfile

```diff
@@ -5,7 +5,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        dnsviz dnsviz 0.9.3 v
+github.setup        dnsviz dnsviz 0.10.0 v
 revision            1
```

Comment:
the "revision" should be reset to 0 when updating the version

Reply:
Fixed in [218df0c](https://github.com/macports/macports-ports/pull/24833/commits/218df0cb93fbfda1ad846592f29fa1b6049ff7a0).